Documentation/getting-started.md

```diff
@@ -43,19 +43,27 @@ dex needs a 32 byte base64-encoded key which will be used to encrypt the private
 
 `DEX_KEY_SECRET=$(dd if=/dev/random bs=1 count=32 2>/dev/null | base64)`
 
+
 The dex overlord and workers allow multiple key secrets (separated by commas) to be passed but only the first will be used to encrypt data; the rest are there for decryption only; this scheme allows for the rotation of keys without downtime (assuming a rolling restart of workers).
 
+# Generate a Admin API Secret
+
+The dex overlord has a an API which is very powerful - you can create Admin users with it, so it needs to be protected somehow. This secret is 128 bytes base64 encoded:
```

Comment:
This could maybe use a connecting sentence that describes how a secret might protect the API, and how it is used. eg "it's information that is very hard to guess that can be shared with approved clients"?